sdk/src/main/java/com/bugsnag/android/SessionTracker.java

```diff
@@ -65,7 +66,9 @@
      * @param date the session start date
      * @param user the session user (if any)
      */
-    @Nullable Session startNewSession(@NonNull Date date, @Nullable User user,
+    @Nullable
+    @VisibleForTesting
```

Comment:
This documents that the method has been left package-visible for existing unit tests, and would otherwise be private.